.github/workflows/e2e.yml

```
@@ -0,0 +1,83 @@
+name: e2e
+
+on:
+  workflow_dispatch:
+  workflow_call:
+  push:
+    branches:
+      - feat/init-provider
```

We need to change this before merging.

```suggestion
      - main
```

---

fixed https://github.com/bucketeer-io/openfeature-kotlin-client-sdk/pull/1/commits/79fcbdb0da296cd3cc033607ac50996fdf66bb4a